javascript-modal/main.js

```diff
@@ -0,0 +1,19 @@
+var $openButton = document.querySelector('.openModal');
+var $container = document.querySelector('.container');
+var $modal = document.querySelector('.modal');
+var $modalButton = document.querySelector('.closeModal');
+
+$modal.hidden = true;
```

Comment:
This actually ends up being inline styling (the same as the `style` attribute on HTML elements) and that will mess up specificity in the future. Instead of this, you should create a class named something like `hidden` with just `display: none` as its styling, then add and remove that class as necessary